### ISSUE TITLE
Error run ConfluentSerdesExample

### ISSUE THREAD

**Issue by ChenShuai1981** [Apicurio/apicurio-registry-demo#13]
```
[main] INFO org.apache.kafka.common.utils.AppInfoParser - App info kafka.consumer for consumer-Consumer-ConfluentSerdesExample-1 unregistered
Exception in thread "main" org.apache.kafka.common.errors.SerializationException: Error deserializing key/value for partition ConfluentSerdesExample-0 at offset 0. If needed, please seek past the record to continue consumption.
Caused by: io.apicurio.registry.rest.client.exception.ArtifactNotFoundException: No artifact with ID '3' in group 'null' was found.
	at io.apicurio.registry.rest.client.exception.ExceptionMapper.map(ExceptionMapper.java:33)
	at io.apicurio.registry.rest.client.impl.ErrorHandler.handleErrorResponse(ErrorHandler.java:63)
	at io.apicurio.rest.client.handler.BodyHandler.lambda$toSupplierOfType$1(BodyHandler.java:46)
	at io.apicurio.rest.client.JdkHttpClient.sendRequest(JdkHttpClient.java:190)
	at io.apicurio.registry.rest.client.impl.RegistryClientImpl.getContentByGlobalId(RegistryClientImpl.java:251)
	at io.apicurio.registry.serde.AbstractSchemaResolver.lambda$resolveSchemaByGlobalId$1(AbstractSchemaResolver.java:176)
	at io.apicurio.registry.serde.ERCache.lambda$getValue$0(ERCache.java:132)
	at io.apicurio.registry.serde.ERCache.retry(ERCache.java:171)
	at io.apicurio.registry.serde.ERCache.getValue(ERCache.java:131)
	at io.apicurio.registry.serde.ERCache.getByGlobalId(ERCache.java:111)
	at io.apicurio.registry.serde.AbstractSchemaResolver.resolveSchemaByGlobalId(AbstractSchemaResolver.java:172)
	at io.apicurio.registry.serde.DefaultSchemaResolver.resolveSchemaByArtifactReference(DefaultSchemaResolver.java:109)
	at io.apicurio.registry.serde.AbstractKafkaDeserializer.resolve(AbstractKafkaDeserializer.java:152)
	at io.apicurio.registry.serde.AbstractKafkaDeserializer.deserialize(AbstractKafkaDeserializer.java:101)
	at io.apicurio.registry.serde.AbstractKafkaDeserializer.deserialize(AbstractKafkaDeserializer.java:127)
	at org.apache.kafka.clients.consumer.internals.Fetcher.parseRecord(Fetcher.java:1386)
	at org.apache.kafka.clients.consumer.internals.Fetcher.access$3400(Fetcher.java:133)
	at org.apache.kafka.clients.consumer.internals.Fetcher$CompletedFetch.fetchRecords(Fetcher.java:1617)
	at org.apache.kafka.clients.consumer.internals.Fetcher$CompletedFetch.access$1700(Fetcher.java:1453)
	at org.apache.kafka.clients.consumer.internals.Fetcher.fetchRecords(Fetcher.java:686)
	at org.apache.kafka.clients.consumer.internals.Fetcher.fetchedRecords(Fetcher.java:637)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1303)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1237)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1210)
	at io.apicurio.registry.examples.confluent.serdes.ConfluentSerdesExample.main(ConfluentSerdesExample.java:125)
```

**Comment by carlesarnal** [comment]
Hey @ChenShuai1981 it would be great if you can provide some context here other than just a stacktrace. Some things like your current config, or even a reproducer are very helpful. In any case, [here](https://github.com/Apicurio/apicurio-registry-examples/blob/master/confluent-serdes/src/main/java/io/apicurio/registry/examples/confluent/serdes/ConfluentSerdesExample.java) you have an example of how to use apicurio-registry with confluent serdes.

**Comment by ChenShuai1981** [comment]
As the comment of ConfluentSerdesExample saying, I launched kafka server in localhost:9092 and Apicurio Registry with docker in localhost:8080. Other examples like simple json/avro/protobuf produce and consume kafka message worked as expected. But when I tried ConfluentSerdesExample, it raised above error. It seems the consumer can NOT get the schema with correct global id. In my case, the correct global id is 8 and content id is 3. But the example looks for global id with 3.

**Comment by ChenShuai1981** [comment]
Btw, How to get ALL groups with Apicurio Registry Rest Client?

**Comment by carlesarnal** [comment]
I did see your question about groups and I will respond on the other one. About the confluent thing, the example is just a very basic demonstration, so depending on your environment or if you modify your configuration, it might not work. That's said, thanks to your issue, I think there's probably a bug on the example, and I have opened an issue on the appropriate repo. https://github.com/Apicurio/apicurio-registry-examples/issues/24 I'm closing this one since this is an outdated repo with no relation with this issue.
